packages/xo-server/src/xo-mixins/proxies.js

```diff
@@ -16,6 +16,33 @@ import patch from '../patch'
 import readChunk from '../_readStreamChunk'
 import { generateToken } from '../utils'
 
+// networks field contains:
+// - <n>/ip <=> <n>/ipv4/0
+// - <n>/ipv4/<m>
+// - <n>/ipv6/<m>
+// where n is the VIF order and m is the IP order
+// for more information, see: https://github.com/xapi-project/xen-api/blob/324bc6ee6664dd915c0bbe57185f1d6243d9ed7e/ocaml/xapi/xapi_guest_agent.ml#L59-L81
+const IPV4_KEY_RE = /^\d+\/ipv4\/\d+$/
+const IPV6_KEY_RE = /^\d+\/ipv6\/\d+$/
+const extractIp = networks => {
```

Comment:
Maybe add a small comment explaining what this function returns?

Reply:
Already done here https://github.com/vatesfr/xen-orchestra/pull/4928/files#diff-03fa9b1c603d4efbb3b28c822269cd8dR32

Reply:
Move it above the function then.